src/finder/frameworkSpecs.ts

```diff
@@ -26,82 +26,82 @@ export const frameworkSpecs: FrameworkSpec[] = [
     ],
     commands: {
       build: {
-        cmd: "ng build",
+        cmd: "npm run build",
       },
       dev: {
-        cmd: "ng run",
+        cmd: "npm run start",
         env: { NODE_ENV: "dev" },
       },
       run: {
-        cmd: "ng run",
+        cmd: "node server.js",
         env: { NODE_ENV: "production" },
       },
     },
   },
   {
-    id: "core:nextjs",
+    id: "nextjs",
     runtime: "nodejs",
-    requiredFiles: [["next.config.js", "next.config.ts"]],
+    requiredFiles: [["next.config.js"]],
     requiredDependencies: [{ name: "next" }],
   },
   {
-    id: "core:astro",
+    id: "astro",
     runtime: "nodejs",
     requiredFiles: [["astro.config.mjs", "astro.config.cjs", "astro.config.js", "astro.config.ts"]],
     requiredDependencies: [{ name: "astrojs" }],
-    embedsFrameworks: ["core:svelte", "core:react", "core:vite"],
+    embedsFrameworks: ["svelte", "react", "vite"],
     commands: {
       build: {
         cmd: "astro build",
       },
       dev: {
-        cmd: "astro preview",
+        cmd: "astro",
         env: { NODE_ENV: "dev" },
       },
       run: {
-        cmd: "astro deploy",
+        cmd: "node ./dist/server/entry.mjs",
         env: { NODE_ENV: "production" },
       },
     },
   },
   {
-    id: "core:react",
+    id: "react",
     runtime: "nodejs",
     requiredDependencies: [{ name: "react" }, { name: "react-dom" }],
   },
   {
-    id: "core:react-vite",
-    runtime: "core:vite",
+    id: "react-vite",
+    runtime: "vite",
```

Comment:
For the initial draft, strongly feel that we don't need the *-vite family.

Reply:
I will keep away some frameworks for the initial drafts and work on discovery logic.